src/measure_theory/lebesgue_measure.lean

```diff
@@ -378,6 +378,74 @@ begin
   simpa [-mem_Ioo] using hx.1.trans hx.2
 end
 
+section region_between
+
+open_locale classical
+
+variable {α : Type*}
+
+/-- The region between two real-valued functions on an arbitrary set. -/
+def region_between (f g : α → ℝ) (s : set α) : set (α × ℝ) :=
```

Comment:
Not completely convinced by the name. `region_between_graphs` is clearer, but not very convincing either.